src/extension/VSProject.cs

```diff
@@ -145,33 +146,33 @@ public TestPackage GetTestPackage(string configName)
         /// </summary>
         public string Name
         {
-            get { return Path.GetFileNameWithoutExtension( ProjectPath ); }
+            get { return Path.GetFileNameWithoutExtension(ProjectPath); }
         }
 
         #endregion
 
         #region Public Methods
 
-        public static bool IsProjectFile( string path )
+        public static bool IsProjectFile(string path)
         {
             if (path.IndexOfAny(Path.GetInvalidPathChars()) >= 0)
                 return false;
 
-            if ( path.ToLower().IndexOf( "http:" ) >= 0 )
-                return false;
-        
-            string extension = Path.GetExtension( path );
+            if (path.ToLower().IndexOf("http:") >= 0)
+                return false;

```

Comment:
I think I'd prefer to ensure this situation never arises in the first place. Can you describe some ways in which it arises in practice?

If we __do__ have to check for this, I'd prefer to recognize the path as a URI and use methods of the `Uri` class rather than string manipulation. If it's any URI besides a file URI, we shouldn't be trying to handle it.

Reply:
@CharliePoole The http check was [added by you](https://github.com/gavardev/vs-project-loader/blame/6cf9ec3279ac6d6f1acab670f3044a4839e6447f/src/extension/VSProject.cs#L171) in the initial upload of this file.  Despite the diff, I don't think it is relevant to @gavardev's PR. I have no idea why someone might pass an http url to this method.

Reply:
##### Re: https://github.com/nunit/nunit3-vs-adapter/issues/438#issuecomment-359593619

> That's a little tacky to point out! 😸
> 
> The "initial upload" wasn't that initial. It was just copying the code that was already present from years back into a new repository. It's possible that the code was needed in it's old context and I'm asking whether it gets any use now, in its present context.
> 
> But I'd think it's a valid question even if the original code had been written by some earlier and dumber version of myself.

Well I apologize for the tackiness! I was attempting to refer to it as an initial upload
_from_ some other repository, in contrast to something we need to ask this contributor. It's a good question and I'm ignorant as to the answer without finding the original repository.

Reply:
Of course, the original repository is NUnitV2 and we could look there. But I think the question that's more relevant is whether the situation still arises. It very well may for certain types of projects if the user points to them. I wonder if a silent return is really that helpful, however.

That said, since it's not an actual change, I don't think we need to hold this PR up for it.